pkg/metrics/metricskey/constants.go

```diff
@@ -47,6 +47,9 @@ const (
 	// EventType is the label for the CloudEvents type context attribute.
 	EventType = "event_type"
 
+	// EventSource is the label for the CloudEvents source context attribute.
+	EventSource = "event_source"
```

Comment:
all these constants are in `knative.dev/pkg/metrics/metricskey` now, as LabelEventSource, etc. Can we use those instead? I'll make the changes in the other places, and remove these constants

Reply:
fixed it